devsetup/standalone/openstack.sh

```diff
@@ -92,7 +93,108 @@ if [ "$EDPM_COMPUTE_CEPH_ENABLED" = "true" ] ; then
     CEPH_ARGS=${CEPH_ARGS:-"-e ~/deployed_ceph.yaml -e /usr/share/openstack-tripleo-heat-templates/environments/cephadm/cephadm-rbd-only.yaml"}
     ENV_ARGS+=" ${CEPH_ARGS}"
 fi
+
+# NOTE: For other nodes than node0, deploy it as extra Heat stacks,
```

Comment:
these conditional blocks will be moved into different make standalone_stack targets